packages/checkout-ui-extensions/src/components/Banner/README.md

```diff
@@ -1,13 +1,9 @@
 # Banner
 
-Use banners to communicate important messages to customers in a prominent way.
+A Banner is used to give feedback and is typically displayed at the top of a page or section.
 
 ## Props
-optional = ?
-
-| Name | Type | Description |
-| --- | --- | --- |
-| title? | <code>string</code> | Banners have an optional title. Use a title to grab the buyers attention with a short, concise message.  |
-| status? | <code>"info" &#124; "success" &#124; "warning" &#124; "critical"</code> | Sets the status of the banner. Default value: <code>'info'</code> |
-| collapsible? | <code>boolean</code> | Makes the content collapsible  |
-| iconHidden? | <code>boolean</code> | Hides the status icon  |
\ No newline at end of file
+
+required = \*
+
+<table><tr><th>Name</th><th>Type</th><th>Description</th></tr><tr><td>title</td><td><code>string</code></td><td>Banners have an optional title. Use a title to grab the buyers attention with a short, concise message.</td></tr><tr><td>status</td><td><code>"info"</code> | <code>"success"</code> | <code>"warning"</code> | <code>"critical"</code></td><td>What status icon & color scheme to use</td></tr><tr><td>collapsible</td><td><code>boolean</code></td><td>Makes the content collapsible</td></tr><tr><td>iconHidden</td><td><code>boolean</code></td><td>Hides the status icon</td></tr></table>
```

Comment:
Do you know why we changed the tables to be half markdown, half HTML?

Reply:
JF helped me figure out that I needed to run `yarn docs:checkout` to update the readmes (that fixed this readme and all others). I will document this!